lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -44,9 +45,19 @@ def get_blocks(self, blocks):
         }
 
 
+class DatesWidgetSerializer(serializers.Serializer):
+    """
+    Serializer for Dates Widget data
+    """
+    course_date_blocks = DateSummarySerializer(many=True)
+    dates_tab_link = serializers.CharField()
+    user_timezone = serializers.CharField()
```

Comment:
Is that not in the general metadata API? Maybe it should be? We also use this in the dates tab API. And might in any tab that presents a time.

Like, the outline tab might want this outside of the dates widget yah, to display the time of a welcome message or course update text or something?